index.js

```diff
@@ -381,7 +382,8 @@ walletRouter.post('/send-tx',
         options: {
           min: 1
         }
-      }
+      },
+      toInt: true
     },
     inputs: {
       in: ['body'],
```

Comment:
Don't we have to convert the `input[].index` to int as well?

Reply:
Makes sense. Because of the way I had to validate the inputs, the conversion was a bit different.

Reply:
Ok. Great!